includes/multi-currency/Compatibility/WooCommerceSubscriptions.php

```diff
@@ -398,7 +398,7 @@ public function maybe_clear_current_my_account_subscription( $formatted, $subscr
 	 *
 	 * @return string The wc_price with HTML wrapping, possibly with the currency code added for explicit formatting.
 	 */
-	public function maybe_get_explicit_format_for_subscription_total( $html_price, $price, $args, $unformatted_price, $original_price ): string {
+	public function maybe_get_explicit_format_for_subscription_total( $html_price, $price, $args, $unformatted_price, $original_price = '' ): string {
```

Comment:
(minor) since the `$price, $args, $unformatted_price, $original_price` arguments are not used, could they be removed from here and from where we register the filter, so the filter registration becomes:
```php
add_filter( 'wc_price', [ $this, 'maybe_get_explicit_format_for_subscription_total' ], 50, 1 );
```

?

Or is this method invoked directly somewhere else (besides the tests)?

Reply:
I think we need to have those 5 arguments, because WooCommerce can call the action with all of them, as [seen here](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-formatting-functions.html#source-view.629).

Reply:
Correct, WC Core can provide them all. But I don't see them used in our function's body. The last argument of `add_filter` is to instruct WordPress to provide the filter a certain number of arguments.
If the last argument is `1`, our function will only be called with the first argument, and we can remove `price, $args, $unformatted_price, $original_price`.

Reply:
Thanks, I learned something new today. Updated in 746fa1134.